README.md

```diff
@@ -1,4 +1,4 @@
-# Outline SDK (Alpha)
```

Comment:
I think it’s helpful to have an indication of the stage in the title

Reply:
added `(Alpha)` back